microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -629,8 +630,24 @@ See also:
 * [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 133.
-* This checkbox is not present in Microsoft Edge Stable 131.
+* This checkbox is present in Microsoft Edge Canary 134.
+* This checkbox is resent in Microsoft Edge Stable 132.
+
+
+<!-- ====================================================================== -->
+## Performance panel: enable third party depenedency<!-- typo in ui --> features
+
+_new as of Microsoft Edge Canary 134_
+
+In the **Performance** tool, enables third-party dependency features.
+<!-- todo: desc:  ", such as **x** **y** **z** controls." -->
```

Comment:
What are the 3rd-party dependency features within the Perf tool?  What are the label & type of controls?
@captainbrosset

Reply:
Enables a new checkbox in the Performance tool to "dim" the performance events that come from third-party dependencies. For example, if your code uses the React library, checking that box will grey-out/dim the React events in a profile.

![image](https://github.com/user-attachments/assets/d00ef8a0-a5ea-4586-8364-7be0c56c88e4)

Reply:
result:
```
Includes a **Dim 3rd Parties** checkbox in the **Performance** tool to "dim" the performance 
events that come from third-party dependencies.  For example, if your code uses the React 
library, checking the **Dim 3rd Parties** checkbox greys-out and dims the React events in a profile.
```